--- /ROPgadget/ropgadget/core.py.original       2019-06-03 21:18:08.781793100 +0000
+++ /ROPgadget/ropgadget/core.py        2019-06-03 21:20:06.045528100 +0000
@@ -12,6 +12,7 @@
 import codecs
 import ropgadget.rgutils as rgutils
 import sqlite3
+import binascii

 from ropgadget.binary             import Binary
 from capstone                     import CS_MODE_32
@@ -108,8 +109,8 @@
             vaddr = gadget["vaddr"]
             insts = gadget["gadget"]
             bytes = gadget["bytes"]
-            bytesStr = " // " + bytes.encode('hex') if self.__options.dump else ""
-
+            bytesStr = " // " + binascii.hexlify(bytes).decode('utf8') if self.__options.dump else ""
+
             print(("0x%08x" %(vaddr) if arch == CS_MODE_32 else "0x%016x" %(vaddr)) + " : %s" %(insts) + bytesStr)

         print("\nUnique gadgets found: %d" %(len(self.__gadgets)))